plexpy/datafactory.py

```diff
@@ -589,6 +589,38 @@ def get_home_stats(self, time_range='30', stats_type=0, stats_count='5', stats_c
                 home_stats.append({'stat_id': stat,
                                    'rows': last_watched})
 
+            elif stat == 'most_concurrent':
+                try:
+                    query = 'SELECT started, stopped ' \
+                            'FROM session_history '
```

Comment:
I was going for all time, but I guess to keep it consistent with the rest of the stats....